bin/setup

```diff
@@ -7,19 +7,19 @@ mkdir -p build/
 if [ ! -d build/go_bootstrap ]; then
   GOOS=$(uname | tr '[:upper:]' '[:lower:]')
   GOARCH=amd64
-  BOOTSTRAP_URL=https://storage.googleapis.com/golang/go1.6.3.$GOOS-$GOARCH.tar.gz
+  BOOTSTRAP_URL=https://storage.googleapis.com/golang/go1.7.5.$GOOS-$GOARCH.tar.gz
```

Comment:
Just a thought. Since you're bootstrapping with a released version of Go anyway (rather than building yourself from source, starting with Go 1.4, etc.), you might as well bootstrap with the latest version of Go (i.e., 1.8.1). It's completely valid to do that. Is there any reason not to?